csrc/scheduler/registry_utils.cpp

```diff
@@ -389,12 +389,35 @@ bool requiresForwardViewReplay(Fusion* fusion, ComputeAtMap& ca_map) {
   return false;
 }
 
+namespace {
+
+bool isSplitOnly(ViewOp* view_op) {
+  for (auto expr : StmtSort::getExprsTo(
```

Comment:
This is fine now but may not be in the future. See #2612. Let's explicitly use the root domains.